Agile0.cpp

```diff
@@ -1,28 +1,32 @@
-#include <iostream>
-#include<Windows.h>
-using namespace std;
-
-void name_1()
-{
-cout << "Anish Khajuria \n" << endl;
-
-
-
-}
-
-
-   
-   
-
-
-int main()
-{
-
-
-name_1;
-
-
-
-
-return 0;
-}
+#include <iostream>
+#include<Windows.h>
+using namespace std;
+
+void name_1()
+{
+cout << "Anish Khajuria \n" << endl;
+
+
+
+}
+
+void printMyName(){
+	cout << "Matthew McCullough\n\n";
+	return;
+}
+   
+   
+
+
+int main()
+{
+
+
+name_1();
+printMyName();
+
+
+
+

```

Comment:
Whoever is the last person to output their names just leave a little Thank you for using our program, or something along those lines